vic/drivers/shared_image/src/vic_init.c

```diff
@@ -1219,22 +1226,12 @@ vic_init(void)
             }
         }
 
-        // Bare soil is now read in as the "last" (highest index) vegetation
-        // class
-        // rescale vegetation classes to 1.0 if their sum is greater than 0.99
-        // otherwise throw an error
-        // TBD: Need better check for equal to 1.
-        if (veg_con[i][0].Cv_sum != 1.) {
+        // Bare soil is now read in as the "last" (highest index) vegetation class
```

Comment:
VIC might add any unclaimed area as a final tile and give that tile bare soil attributes, yes.  But again, if someone uses a veg library that has an explicit bare soil class, that class could be anywhere in the order, and VIC wouldn't know or care.  All I'm saying is that the comments shouldn't give people the impression that bare soil is only ever stored in the tile having index = Nveg.